pom.xml

```diff
@@ -58,10 +58,43 @@
       <version>1.9.80</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>3.4.4</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+    <dependency>
+			<groupId>org.powermock</groupId>
```

Comment:
I am not sure what happened to formatting here, it is fine on my end will look into it :(